migrations/915-rename-addon-tags.sql

```diff
@@ -0,0 +1 @@
+RENAME TABLE `users_tags_addons` TO `addons_tags`;
```

Comment:
WRT testing... I'm comfortable with the dev/stage roll-out to test any thing that may break.